managedtenants/bundles/csv.py

```diff
@@ -0,0 +1,58 @@
+import yaml
+
+from managedtenants.bundles.exceptions import CSVError
+
+
+class CSV:
+    def __init__(self, manifests_dir, operator_name):
+        self.manifests_dir = manifests_dir
+        self.operator_name = operator_name
+        self.path = self._get_path()
+        self.data = self._parse_csv()
+
+    def _get_path(self):
+        """
+        Find the csv manifest path.
+        """
+        candidates = [
+            f"{self.operator_name}.csv.yaml",
+            f"{self.operator_name}.clusterserviceversion.yaml",
```

Comment:
I've noticed that some addon teams use `.yml` extensions as well. Do you think its a better idea to use regex here?
(I hate regex myself :D)

Reply:
Thanks for spotting this. Refactored to a double for loop.

Reply:
Re-refactored to only implement suffixes, there are too many name differences.